phpBB/includes/acp/acp_profile.php

```diff
@@ -25,6 +25,10 @@ class acp_profile
 
 	var $edit_lang_id;
 	var $lang_defs;
+
+	/**
+	 * @var \phpbb\di\service_collection
+	 */
 	protected $type_collection;
```

Comment:
If I use `ArrayAccess` or `ArrayObject` the method `service_collection#add()` won't be found by the IDE as it is added by phpBB.